### ISSUE TITLE
Add an option to use a custom Favicon

### ISSUE THREAD

**Issue by kayvanaarssen** [pglombardo/PasswordPusher#1597]
Can you make it an option in the config file to use a custom favicon.
Either link to one, or link to a location inside the docker container (mount location)

**Comment by pglombardo** [comment]
Hi @kayvanaarssen - you can configure the favicons with [these settings](https://github.com/pglombardo/PasswordPusher/blob/master/config/settings.yml#L591-L625).  Make sure to set the `icon_57x57` to something as the customisation is [keyed off of that value](https://github.com/pglombardo/PasswordPusher/blob/master/app/views/shared/_layout_icons.html.erb#L2).

**Comment by kayvanaarssen** [comment]
Thanks that did the Trick, but I did link it like this: 

```
- type: bind
        source: './custom/favicon'
        target: '/opt/PasswordPusher/app/assets/images/favicon'
```

Than in the `.env` i've set the path like: `/opt/PasswordPusher/app/assets/images/favicon/favicon.png`

It did not like that...
So for now i've just linked them to our public website to grab the Favicon but is this a bug or am I doing something wrong here?

**Comment by pglombardo** [comment]
Yeah I should probably add a note about that.  This is complexity with the app framework.  If you place a file in `assets/images`, then assets need to be precompiled, fingerprinted and the asset needs to be loaded with `asset_path`.

We don't do that for custom icons - we just put the raw value on the page so it has to be a fully qualified HTTPS URL.

**Comment by kayvanaarssen** [comment]
Right, did not know that😉 Not using the pre compile option so that's why it did not work.
Now loading it directly from our website so that's also fine.

**Comment by pglombardo** [comment]
HI @kayvanaarssen - I'll close this out soon if nothing remains.  BTW I saw the other issue you filed - I'll have a closer look at it over the weekend.

**Comment by pglombardo** [comment]
Thanks for the suggestion — this has been available for a while now.

You can set a custom favicon (and the related mobile icons) in settings or via env vars, either as a path inside the container or as a full URL:

`PWP__BRAND__ICON_16x16`, `PWP__BRAND__ICON_32x32`, `PWP__BRAND__ICON_57x57`, `PWP__BRAND__ICON_96x96`, and the rest of the `PWP__BRAND__ICON_*` family.

At a minimum, set `icon_57x57` and `icon_96x96`.

If anything remains, please let us know. We can re-open if needed.
